tests/InputFiles/ScalarWave/SphericalWave3DPeriodic.yaml

```diff
@@ -0,0 +1,54 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveRegularSphericalWave3D
+# Check: parse;execute
+
+AnalyticSolution:
+  RegularSphericalWave:
+    Profile:
+      Gaussian:
+        Amplitude: 1.0
+        Width: 1.0
+        Center: 0.0
+
+Evolution:
+  InitialTime: 0.0
+  InitialTimeStep: 0.001
+  InitialSlabSize: 0.01
+  TimeStepper:
+    AdamsBashforthN:
+      Order: 3
+  StepController: BinaryFraction
+  StepChoosers:
+    - Constant: 0.05
+    - Increase:
+        Factor: 2
+    - Cfl:
+        SafetyFactor: 0.2
+
+DomainCreator:
+  Brick:
+    LowerBound: [0.0, 0.0, 0.0]
+    UpperBound: [6.0, 6.0, 6.0]
+    IsPeriodicIn: [true, true, true]
```

Comment:
Does it make sense to have this domain be periodic? How about you make the domain a Sphere for this test?

Reply:
Agreed, changed to spherical domain with dirichlet BCs